flashinfer/moe_ep/kernel_src/cutedsl_megamoe/src/moe_nvfp4_swapab/contract.py

```
@@ -81,7 +82,7 @@ def linearize(self, coord: Sequence[int]) -> int:
 
         linear = 0
         stride = 1
-        for axis, (idx, size) in enumerate(zip(coord_tuple, self.sizes, strict=False)):
+        for axis, (idx, size) in enumerate(zip(coord_tuple, self.sizes)):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)